arkouda/pdarrayclass.py

```diff
@@ -115,7 +140,32 @@ def format_other(self, other : object) -> object:
         return fmt.format(other)
 
     # binary operators
-    def binop(self, other : object, op : str):
+    def binop(self, other : 'pdarray', op : str) -> 'pdarray':
```

Comment:
`binop` is not suppose to be user facing. maybe it should get underscores prepended to the name?

Reply:
Agree

Reply:
fixed!